roles/setup/tasks/main.yml

```diff
@@ -1,72 +1,74 @@
 - debug:
-    msg: 'Preparing Keycloak setup ...'
-  name: 'Get started'
+    msg: "Preparing Keycloak setup ..."
+  name: "Get started"
 - include_vars:
     file: catalog_rbac.yml
     name: rbac
-  name: 'Include Catalog & Approval RBAC Setup vars'
+  name: "Include Catalog & Approval RBAC Setup vars"
 - community.general.keycloak_realm:
-    auth_client_id: '{{ auth_client_id }}'
-    auth_keycloak_url: '{{ auth_keycloak_url }}'
-    auth_password: '{{ auth_password }}'
-    auth_realm: '{{ auth_realm }}'
-    auth_username: '{{ auth_username }}'
-    display_name: 'Ansible Automation Platform'
+    auth_client_id: "{{ auth_client_id }}"
+    auth_keycloak_url: "{{ auth_keycloak_url }}"
+    auth_password: "{{ auth_password }}"
+    auth_realm: "{{ auth_realm }}"
+    auth_username: "{{ auth_username }}"
+    display_name: "Ansible Automation Platform"
     enabled: true
     attributes:
-      frontendUrl: '{{ realm_frontend_url }}'
-    id: '{{ realm_name }}'
-    realm: '{{ realm_name }}'
+      frontendUrl: "{{ realm_frontend_url }}"
+    id: "{{ realm_name }}"
+    realm: "{{ realm_name }}"
     state: present
-    validate_certs: true
-  name: 'Create or update AAP Keycloak realm'
+    validate_certs: "{{ verify_ssl }}"
```

Comment:
@Alex-Izquierdo According to the ansible module for keycloak this only supports boolean data. https://docs.ansible.com/ansible/latest/collections/community/general/keycloak_client_module.html#parameter-validate_certs

Reply:
I don't understand this comment. The variable takes already a boolean.